### PR TITLE
Requested balance changes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Increases 6.5, 408, and 7.5 damages a small bit to make them feeeel more impactful.
gives 5 more damage to 10mm caseless

### CODE REVIEW

**Comment by Lamasmaster** [comment]
You have now managed to make pistol calibers completely worthless, truly sojourn moment

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 1]
```diff
@@ -591,7 +591,7 @@
 
 /obj/item/projectile/bullet/heavy_rifle_408/lethal
 	name = "hollow-point bullet"
-	damage_types = list(BRUTE = 20)
+	damage_types = list(BRUTE = 24)
 	armor_divisor = 0.7
 	wounding_mult = WOUNDING_WIDE
```
Comment (Dimasw99) [inline code/modules/projectiles/projectile/bullettypes.dm:596]: At 0 armor: FMJ 30 * 1.5 = 45dmg
HP: 24 * 1.5 = 36 dmg

Might want to adjust HP to be better than FMJ against unarmored

Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:596]:
```suggestion
	wounding_mult = WOUNDING_WIDE
```

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 2]
```diff
@@ -510,7 +510,7 @@
 
 /obj/item/projectile/bullet/rifle_75/lethal
 	name = "hollow-point bullet"
-	damage_types = list(BRUTE = 19)
+	damage_types = list(BRUTE = 21)
 	armor_divisor = 0.65
 	wounding_mult = WOUNDING_WIDE
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:515]:
```suggestion
	wounding_mult = WOUNDING_EXTREME
```

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 3]
```diff
@@ -409,7 +409,7 @@
 
 /obj/item/projectile/bullet/light_rifle_257/lethal
 	name = "hollow-point bullet"
-	damage_types = list(BRUTE = 17)
+	damage_types = list(BRUTE = 19)
 	armor_divisor = 0.5
 	wounding_mult = WOUNDING_SERIOUS
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:414]:
```suggestion
	wounding_mult = WOUNDING_WIDE
```

**Pull request merged**